secure_message/authentication/jwe.py

```diff
@@ -10,21 +10,25 @@
 from cryptography.hazmat.primitives.ciphers import modes
 from cryptography.hazmat.primitives.asymmetric import padding
 from cryptography.hazmat.primitives import hashes
-from app import settings
+from flask import json, current_app
 from werkzeug.exceptions import BadRequest
-from flask import json
 
 IV_EXPECTED_LENGTH = 12
 CEK_EXPECT_LENGTH = 32
 
 
 class Encrypter:
 
-    def __init__(self, _private_key=settings.SM_USER_AUTHENTICATION_PRIVATE_KEY,
-                 _private_key_password=settings.SM_USER_AUTHENTICATION_PRIVATE_KEY_PASSWORD,
-                 _public_key=settings.SM_USER_AUTHENTICATION_PUBLIC_KEY):
+    def __init__(self, _private_key=None, _private_key_password=None, _public_key=None):
         """ initialise encrypter with correct keys, the ability to change default keys for tests"""
 
+        if _private_key is None:
+            _private_key = current_app.config['SM_USER_AUTHENTICATION_PRIVATE_KEY']
```

Comment:
Could be `_private_key  = _private_key  or current_app.config['SM_USER_AUTHENTICATION_PRIVATE_KEY']`